plugins/power/gsd-power-constants.h

```diff
@@ -18,7 +18,7 @@
  */
 
 /* The blank delay when the screensaver is active */
-#define SCREENSAVER_TIMEOUT_BLANK                       15 /* seconds */
+#define SCREENSAVER_TIMEOUT_BLANK                       30 /* seconds */
```

Comment:
Could this change be upstreamed? The rationale seems widely applicable.

Reply:
https://gitlab.gnome.org/GNOME/gnome-settings-daemon/merge_requests/100